java-client/src/test/java/org/scassandra/http/client/ActivityClientTest.java

```diff
@@ -112,11 +112,11 @@ public void testErrorDuringConnectionRetrieval() {
         //then
     }
 
-    @Test(expected = ActivityRequestFailed.class, timeout = 2500)
+    @Test(expected = ActivityRequestFailed.class, timeout = 10000)
```

Comment:
I increased the base timeouts in the clients from 1 to 5 seconds as I was seeing that travis was a bit finicky from time to time with actual timeouts occurring when we didn't want them to.  In the interest of expedience, maybe I can tune the timeout lower just for those tests so they don't wait 5 secs a piece.